section4/reflection.md

```diff
@@ -2,20 +2,31 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
-1. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+  It helped my work feel more focused and efficient. During the times I set aside to put in work, I was more deliberate with making progress.
 
-1. What is an Object, and how is it different from an Array in Javascript?
+2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
-1. For each set of data, would an array or object be better to store it? Explain your choice.
+  I thought I was more focused! I would like to try using a mechanical timer, so I get the added pressure of the ticking sound counting down the allotted time.
 
-  * List of all of the students in class
-  * List of states and their capitals
-  * List of things to pack for vacation
-  * Names of all the Instagram accounts I follow
-  * List of student names and their cohort
-  * Ingredients and amount of each ingredient to bake a cake
-  * All my favorite restaurants
+3. What is an Object, and how is it different from an Array in Javascript?
 
-1. In this section, we talked about an `transaction item object`. It has keys that represent different properties an bank transaction would have, and values that store the data. In a banking app, this data would eventually be displayed in the browser for the user. Think of another Object in a web application that you use frequently. What is it? What keys might be on it? What are example values? 
+  An object is something that has individual properties and can have methods as well. In addition, objects contain arrays of properties. An array is an ordered list of values.
```

Comment:
🚨To help you with your definitions
Something with individual properties and methods is called a C_______
Something that has key/value pairs is called an O_______